R/symComp.R

```diff
@@ -0,0 +1,89 @@
+# symComp.R
+
+#'
+#' \code{symComp} Return a list of domains that the genes in two systems
+#'    overlap along with genes that mapped to the same domains
+#'
+#' @param system1 The name of the system in interest. Is compared with system2
```

Comment:
The @param and @return need to be changed according to the format described here:
https://github.com/hyginn/BCB420.2019.ESA/blob/master/PullRequestReviewNotes.md

The format is: mention the type and dimension of parameters and return values in the header. Type is one of {numeric|integer|double|complex|logical|character|function} ... or a mixed type such as {data frame|list}. The dimension is one of {a|a vector|a matrix|a 3D matrix|...},
For example:
@return (numeric)  A vector of the next six 6-of-49 lottery numbers or NULL.